src/config/dark-sites.config

```diff
@@ -516,6 +517,7 @@ sufficientvelocity.com
 sumo.app
 sunxdcc.com
 supinic.com
+support.pluto.tv
```

Comment:
Subdomains are automatically targeted trough `pluto.tv` you can remove this entry

Reply:
oh ok